### PR TITLE
Angle style param should respect numeric value from JS Function

### PR DESCRIPTION
**Pull request by tallytalwar**
Refer discussion https://github.com/tangrams/tangram-docs/issues/250

### CODE REVIEW

File: core/src/scene/styleContext.cpp [hunk 1]
```diff
@@ -487,6 +487,11 @@ void StyleContext::parseStyleResult(StyleParamKey _key, StyleParam::Value& _val)
                 _val = static_cast<uint32_t>(duk_get_uint(m_ctx, -1));
                 break;
             }
+            case StyleParamKey::angle: {
+                std::string value = doubleToString(static_cast<double>(duk_get_number(m_ctx, -1)));
+                _val = StyleParam::parseString(_key, value);
```
Comment (matteblair) [inline core/src/scene/styleContext.cpp]: Can't we just assign the double directly to the parameter value?

Comment (tallytalwar) [reply core/src/scene/styleContext.cpp]: Ohh yes, since `auto` return will be handled by `duk_is_string`.

Comment (tallytalwar) [reply core/src/scene/styleContext.cpp]: Updated!

Comment (tallytalwar) [reply core/src/scene/styleContext.cpp]: K actually fixed now!

**Pull request merged**